src/components/FileVideo.tsx

```diff
@@ -76,13 +79,12 @@ const FileVideo: FC<FileVideoProps> = ({
     if (!isPlayerView) {
       navigation.setOptions({
         headerRight: () => (
-          <View>
+          <View style={styles.headerButtons}>
+            <ChatButton item={item} />
             {isDownloading ? (
-              <View style={styles.headerButtonsDownloadingState}>
-                <FileHeaderButton disabled={true} name="cloud-download" />
-              </View>
+              <FileHeaderButton disabled={true} name="cloud-download" />
```

Comment:
Wouldn't that logic be useful for all download button? Consider a `DownloadButton` component to reuse for all files.